tests/test_articles.py

```diff
@@ -554,6 +574,23 @@ def test_update_realname_article(self):
         assert result.get('name_type') == BoardNameType.REALNAME
         assert Article.objects.get(title=new_title).name_type == BoardNameType.REALNAME
 
+    def test_ban_canclellation_vote_after_30(self):
```

Comment:
오타: canclellation → cancellation
cancellation_vote보다 vote_cancellation이 나아 보입니다.
```suggestion
    def test_ban_vote_canclellation_after_30(self):
```